analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.cs

```diff
@@ -98,6 +98,22 @@ End If
         [TestMethod]
         public void Branching_PersistSymbols_BetweenBlocks()
         {
+            var setter = new PreProcessTestCheck(x =>
```

Comment:
Why this change? You could use `PostProcess` instead in `SetTestConstraintCheck`.